<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QListWidget" name="arduino_list">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConnection">
    <property name="title">
     <string>Connection</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
   </widget>
   <widget class="QMenu" name="menuLogs">
    <property name="title">
     <string>Logs</string>
    </property>
    <addaction name="actionGet_Logs"/>
    <addaction name="actionGet_Logs_By_Pin"/>
   </widget>
   <addaction name="menuConnection"/>
   <addaction name="menuLogs"/>
  </widget>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionGet_Logs">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Get Logs</string>
   </property>
  </action>
  <action name="actionGet_Logs_By_Pin">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Get Logs By Pin</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
